recipes/nbtutor/build.sh

```diff
@@ -0,0 +1,19 @@
+conda install --yes nodejs
+# FIXME: This is an ugly hack. nodejs only gets installed and linked in the
+# first build env and doesn't get linked in following build envs
```

Comment:
This is horribly wrong, but for some reason nodejs gets installed and linked in the first conda test env (py27), but does not get linked in any of the following conda envs. I am guessing it has to do with nodejs being a python no arch build.

This hack is purely to get the staged recipe to pass tests and can (will) be removed when its a feedstock as, afaik, the builds get run independently in individual jobs Travis and CircleCI.